spacelift/resource_stack.go

```diff
@@ -505,6 +511,11 @@ func resourceStackDelete(ctx context.Context, d *schema.ResourceData, meta inter
 
 	variables := map[string]interface{}{"id": toID(d.Id())}
 
+	deletionProtection := d.Get("deletion_protection").(bool) 
+	if deletionProtection {
+		return diag.Errorf("Due to deletion_protection, could not delete stack: %v", internal.FromSpaceliftError(err))
```

Comment:
I don't think `err` is populated here?

Reply:
@marcinwyszynski woops, you're right. Bad copy pasta.